### PR TITLE
yamllint formatting cleanup for ocp-workload-optaweb-vrp

### PR DESCRIPTION
**Pull request by ritesh97-rh**
##### SUMMARY
For the specified roles:
- Create a yamllint file to enforce formatting rules
- Make changes to existing code base to adhere to formatting requirements
- yamllint errors will be displayed during formatting checks in future development

##### ISSUE TYPE
- Code Cleanup Request

##### COMPONENT NAME
ocp-workload-optaweb-vrp

### CODE REVIEW

File: ansible/roles/ocp-workload-optaweb-vrp/tasks/wait_for_deploy.yml [hunk 1]
```diff
@@ -15,16 +17,24 @@
   changed_when: false
   retries: "{{ deploy_status_retries }}"
   delay: "{{ deploy_status_delay }}"
-  until: 'rc_state.stdout |from_json | json_query("items[? (kind == ''ReplicationController'')].metadata.annotations.\"openshift.io/deployment-config.name\"") |intersect(item) |length >= 1'
-  loop: "{{pod_to_wait}}"
+  until: >-
+    rc_state.stdout | from_json | json_query("items[?
+    (kind == ''ReplicationController'')].metadata.annotations."openshift.io/deployment-config.name"")
+    | intersect(item) | length >= 1
+  loop: {{pod_to_wait}}
```
Comment (stencell) [inline ansible/roles/ocp-workload-optaweb-vrp/tasks/wait_for_deploy.yml:24]: This didn't get the whitespace around the var and probably needs to be quoted.

File: ansible/roles/ocp-workload-optaweb-vrp/tasks/workload.yml [hunk 2]
```diff
@@ -1,51 +1,54 @@
...
+  shell: "oc get route optaweb-vrp-backend -n {{ ocp_project }} | awk 'FNR > 1 {print $2}'"
   register: oc_get_route_output
 
 - name: "Set OptaWeb backend Route fact"
   set_fact:
     optaweb_vrp_backend_route: "{{ oc_get_route_output.stdout }}"
 
 - name: "Create OptaWeb VRP front-end ReactJS app"
-  shell: "oc new-app --name=optaweb-vrp-frontend -e REACT_APP_BACKEND_URL=http://{{optaweb_vrp_backend_route}} \
+  shell: "oc new-app --name=optaweb-vrp-frontend -e REACT_APP_BACKEND_URL=http://{{ optaweb_vrp_backend_route }} \
```
Comment (stencell) [inline ansible/roles/ocp-workload-optaweb-vrp/tasks/workload.yml:46]: this should be corrected to maintain consistency with other uses of multiline `shell` in this file

**Review by stencell — CHANGES_REQUESTED** [review]
A couple comments to address and this needs a test since there were code changes.

File: ansible/roles/ocp-workload-optaweb-vrp/defaults/main.yml [hunk 3]
```diff
@@ -24,10 +24,12 @@ build_status_delay: 30
...
+#openjdk_imagestreams_yml:
+#https://raw.githubusercontent.com/jboss-openshift/application-templates/master/openjdk/openjdk18-image-stream.json
 openjdk_imagestreams_yml: https://raw.githubusercontent.com/jboss-openshift/application-templates/ose-v1.4.15/openjdk/openjdk18-image-stream.json
 
 optaweb_vrp_git_repository: https://github.com/kiegroup/optaweb-vehicle-routing
 
 #pam_version_tag: 7.3.0.GA
-#pam_imagestreams_yml: https://raw.githubusercontent.com/jboss-container-images/rhpam-7-openshift-image/{{pam_version_tag}}/rhpam73-image-streams.yaml
+#pam_imagestreams_yml:
+#https://raw.githubusercontent.com/jboss-container-images/rhpam-7-openshift-image/{{pam_version_tag}}/rhpam73-image-streams.yaml
```
Comment (jkupferer) [inline ansible/roles/ocp-workload-optaweb-vrp/defaults/main.yml:35]: This comment style does not make sense with the linebreak

File: ansible/roles/ocp-workload-optaweb-vrp/tasks/wait_for_deploy.yml [hunk 4]
```diff
@@ -15,16 +17,24 @@
   changed_when: false
   retries: "{{ deploy_status_retries }}"
   delay: "{{ deploy_status_delay }}"
-  until: 'rc_state.stdout |from_json | json_query("items[? (kind == ''ReplicationController'')].metadata.annotations.\"openshift.io/deployment-config.name\"") |intersect(item) |length >= 1'
-  loop: "{{pod_to_wait}}"
+  until: >-
+    rc_state.stdout | from_json | json_query("items[?
+    (kind == ''ReplicationController'')].metadata.annotations."openshift.io/deployment-config.name"")
```
Comment (jkupferer) [inline ansible/roles/ocp-workload-optaweb-vrp/tasks/wait_for_deploy.yml:22]: This `''` quote is not right.

**Review by jkupferer — CHANGES_REQUESTED** [review]

**Pull request closed without merging**